instructions/section-2/index.html

```diff
@@ -0,0 +1,10 @@
+Stephanie Alvarado Gallegos 
```

Comment:
You might need to go back to the instructions in the README section, HTML is not written in plain text, it has a syntax made up of tags, and you're meant to place these texts inside those tags